cmd/root.go

```diff
@@ -237,6 +237,10 @@ func JsonnetVM(cmd *cobra.Command) (*jsonnet.VM, error) {
 		}
 	}
 
+	resolverType := kubecfg.ResolverType_value[viper.GetString(flagResolver)]
+	resolverFailureAction := kubecfg.ResolverFailureAction_value[viper.GetString(flagResolvFail)]
```

Comment:
It is "weird" to access this kind of value mapping from a different package. I'd create a `Parse` method to parse the flag string and make the internal value mappings "private" (unexported).

#### Extra points

Consider having a method to list the enum's allowed string values and one to get the string of the enum type.

Then update the cmd flag definitions to use the `NoopResolver` string representation as the default flag value and the enum's allowed string values as part of the flag's description.

Reply:
Done!